### PR TITLE
bz4763: Add file filters for the "Open" dialog

### PR DESCRIPTION
**Pull request by bendk**
Big thanks to Ben Sturmfels.

It's nice to fix a 4-digit ticket.

### CODE REVIEW

File: tv/lib/frontends/widgets/application.py [hunk 1]
```diff
@@ -469,7 +469,23 @@ def reveal_conversions_folder(self):
 
     def open_video(self):
         title = _('Open Files...')
-        filenames = dialogs.ask_for_open_pathname(title, select_multiple=True)
+        audio_extensions = [mem.replace(".", "")
```
Comment (paulswartz) [inline tv/lib/frontends/widgets/application.py]: Probably should be `mem[1:]` for all three.

File: tv/lib/filetypes.py [hunk 2]
```diff
@@ -41,6 +41,7 @@
                     '.rmvb', '.mkv', '.m2v', '.ogm', '.webm', '.ogx']
 AUDIO_EXTENSIONS = ['.mp3', '.m4a', '.wma', '.mka', '.flac', '.ogg', '.oga']
 FEED_EXTENSIONS = ['.xml', '.rss', '.atom']
+TORRENT_EXTENSIONS = ['.torrent']
```
Comment (paulswartz) [inline tv/lib/filetypes.py:44]: Don't we have this already?

Comment (bendk) [inline tv/lib/filetypes.py:44]: As far as I can tell, no.

File: tv/lib/frontends/widgets/application.py (hunk 1 above)
Comment (bendk) [inline tv/lib/frontends/widgets/application.py]: Good point.  I'll fix this.

**Comment by paulswartz** [comment]
Do we have a contributor agreement from Ben? If yes, this is good to merge.

**Comment by bendk** [comment]
contributor agreement is in place.

**Pull request merged**